### PR TITLE
Updated foreman_docker to 3.1.0 [rpm]

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.15
* [ ] 1.14
* [ ] 1.13

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

I realize this should probably be 4.0 because it's breaking compatibility with Foreman < 1.15 though..

### CODE REVIEW

File: rubygem-foreman_docker/rubygem-foreman_docker.spec [hunk 1]
```diff
@@ -17,22 +17,24 @@
...
 Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
-Requires:   foreman-compute >= 1.13.0
+Requires:   foreman-compute >= 1.15.0
 Requires:   %{?scl_prefix}rubygem(docker-api) >= 1.18.0
 Requires:   %{?scl_prefix}rubygem(docker-api) < 2.0
 Requires:   %{?scl_prefix}rubygem(wicked) >= 1.1
 Requires:   %{?scl_prefix}rubygem(wicked) < 2.0
 Requires:   %{?scl_prefix}rubygem(deface) < 2.0
+Requires:   %{?scl_prefix}rubygem(excon) >= 0.46
```
Comment (domcleal) [inline rubygem-foreman_docker/rubygem-foreman_docker.spec]: This should also be listed as a BuildRequires, they should match.

**Comment by dLobatog** [comment]
@domcleal Updated to match Requires & BuildRequires

**Comment by dLobatog** [comment]
[test] as koji was borked yesterday evening

**Comment by ehelms** [comment]
[test]

**Pull request merged**